src/BenchmarksApps/TodosApi/TodosApi.json

```diff
@@ -45,7 +45,13 @@
         ],
         "operationId": "CreateTodo",
         "requestBody": {
-          "content": { },
+          "content": {
```

Comment:
For whatever reason, request body schemas weren't populated in the previous version of the document.